### DISCUSSION TITLE
Container for x86 on web, WebContainer?

### DISCUSSION THREAD

**Discussion by dinfer** [copy/v86#709] [General]
Instead of implement a CPU, is it possible to only translate x86 byte code to wasm and add some system API polyfill? WASM hosted by browser maybe faster than js CPU emulator, and if I only want to run "nodejs" or something like that in browser, there is no need to emulate the whole system.

**Comment by Darin755** [comment]
Interesting idea

**Comment by ProgrammerIn-wonderland** [comment]
Sounds like cheerpx
